html/captive-portal/templates/server_error.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fatal error !</title>
+  </head>
+
+  <body>
+    <h2>Fatal error !</h2>
+    <p>A server error has occured, please try again or try to contact your local support staff</p>
```

Comment:
This displays when s**t hits the fan and doesn't go through any templating at all
Since this display when there is an application error, the renderer won't be available.

Adding templating to that would add a possibility of having an error in the error handling

![image](https://cloud.githubusercontent.com/assets/3857942/14115730/3429b308-f5aa-11e5-84e0-2e3b15dc83bf.png)